src/lib/constants.ts

```diff
@@ -1 +1,4 @@
 export const host = process.env.HOST ?? "https://projectprotoco.org";
```

Comment:
```suggestion
export const host = process.env.HOST ?? "https://projectprotocol.org";
```

Like this? I see it was already incorrect; must've slipped by in prior reviews.